### ISSUE TITLE
How to set wordcouns and reading time like your sites？

### ISSUE THREAD

**Issue by kangzhiheng** [puresyntax71/hugo-theme-chunky-poster#100]
Thx your geart work! I like it, but I have some troubles. Like your [site](https://themes.gohugo.io//theme/hugo-theme-chunky-poster/post/markdown-syntax/), How to set **wordcounts** and **reading time**？
![image](https://user-images.githubusercontent.com/30773002/81943424-f23cff80-962d-11ea-91b3-a2e7fb26ef9a.png)

**Comment by puresyntax71** [comment]
Yeah, it's automated or perhaps you're not seeing it on your end? Perhaps check for a file `en.yaml` - https://github.com/puresyntax71/hugo-theme-chunky-poster/blob/master/i18n/en.yaml? It should be working out of the box though.
